### DISCUSSION TITLE
How to disable query trigger in ApolloQuery component and call it programmatically based on some logic.

### DISCUSSION THREAD

**Discussion by jaybharadia** [Akryum/vue-apollo#1470] [Q&A]
Currently, behaviour is that query is hitting as soon as the component mounts.

**Comment by jaybharadia** [comment]
I got the solution
- Use `skip` prop 
- Extract `query` slot scope 
- call `query.start` to programmatically call query.

**Comment by Akryum** [comment]
In `@vue/apollo-composable` you can also use `useLazyQuery`
